File: spigot/src/main/java/com/dreamdisplays/listeners/PlayerListener.java
package com.dreamdisplays.listeners;

import com.dreamdisplays.managers.PlayerManager;
import org.bukkit.event.EventHandler;
import org.bukkit.event.Listener;
import org.bukkit.event.player.PlayerQuitEvent;

public class PlayerListener implements Listener {

    @EventHandler
    public void onPlayerLeave(PlayerQuitEvent event) {
        PlayerManager.removeVersion(event.getPlayer());
    }
}

File: spigot/src/main/java/com/dreamdisplays/listeners/PlayerListener.kt
package com.dreamdisplays.listeners

import com.dreamdisplays.managers.PlayerManager
import org.bukkit.event.EventHandler
import org.bukkit.event.Listener
import org.bukkit.event.player.PlayerQuitEvent

class PlayerListener : Listener {
    @EventHandler
    fun onPlayerLeave(event: PlayerQuitEvent) {
        PlayerManager.removeVersion(event.getPlayer())
    }
}
